### ISSUE TITLE
Namron Zigbee dimmer 400W, Scene/Switch

### ISSUE THREAD

**Issue by olemr** [dresden-elektronik/deconz-rest-plugin#2445]
Hello.
I'm on:
![image](https://user-images.githubusercontent.com/28068256/74483098-0732b700-4eb6-11ea-8938-21be41cd0ec8.png)

Have a question regarding this device:
![image](https://user-images.githubusercontent.com/28068256/74482546-077e8280-4eb5-11ea-8078-441b1b922b4b.png)
[4512700 INSTRUCTION MANUAL.pdf](https://github.com/dresden-elektronik/deconz-rest-plugin/files/4201439/4512700.INSTRUCTION.MANUAL.pdf)

Just added it as a light, and it joined just fine after a power-up.
Can control the light just fine, but if I use the connected push(WAF)-switch turning the light on or off, the "on:" state takes 5-10s to be updated.

![image](https://user-images.githubusercontent.com/28068256/74482820-8d9ac900-4eb5-11ea-884e-bc11ca6dea65.png)

I see in the manual, that the device supports:
![image](https://user-images.githubusercontent.com/28068256/74482912-b28f3c00-4eb5-11ea-90c4-96bd323fe1dc.png)

Would/should it be possible to add a switch/sensor device for the push-button as well, with a quicker response?

**Comment by ebaauw** [comment]
The delay is caused by the REST API plugin polling the lights for their state.  This looks suspiciously like another Sunricher OEM dimmer, so it should support attribute reporting.  This needs to be configured, so the dimmer sends updates on state changes, eliminating the delay.

The dimmer would need to be whitelisted in the code, for the REST API plugin to configure attribute reporting on pairing.  Best configure it manually in the GUI, see the user manual (under _Help_).  Bind the _On/Off_ and _Level Control_ clusters to the coordinator, and configure attribute reporting for _OnOff_ (1/300) and _Current Level_ (1/300/1).

> Would/should it be possible to add a switch/sensor device for the push-button as well, with a quicker response?

No, the connected switch isn't exposed over ZigBee.  Neither is the minimum dim level.

**Comment by olemr** [comment]
Thanks.
Will try this during the weekend.
Will report back.

**Comment by olemr** [comment]
This looks very promising!
Only things that didn't match your description 100% was the cluster names.
![image](https://user-images.githubusercontent.com/28068256/74566388-a5388700-4f73-11ea-93a7-2314a146a431.png)

Thanks for helping out!

**Comment by ebaauw** [comment]
> Only things that didn't match your description 100% was the cluster names.

The screenshot contains the **attribute** names of the _Level Control_ cluster (0x0008).  You need to set the reporting configuration for 0x0000, _Current Level_.  _On Level_ is the default level when you turn on the dimmer; value 255 means the previous value.  The _OnOff_ attribute (also 0x0000) is in the _On/Off_ cluster (0x0006).

> Will the two bindings I just performed be part of a backup? (where is it stored?)

On the dimmer, in non-volatile memory.

**Comment by olemr** [comment]
A bit confusing the first time around.
Just discovered that the config I did yesterday har left the device uncontrollable.
Trying once more :-)

**Comment by olemr** [comment]
After a factory reset, I got it working.
For other newbies (like me) coming here, and who are also a bit unsure even after reading the manual I will recap what I did.
Binding cluster 0006 & 0008 drag them to the Source, and the 01 cluster of the controller to the target. Bind them both:
![image](https://user-images.githubusercontent.com/28068256/74589881-a87e5200-5009-11ea-863b-98420d13c646.png)

For cluster 0006, double click on On/Off and set values 1/300
![image](https://user-images.githubusercontent.com/28068256/74590094-9ac9cc00-500b-11ea-887c-a6646434aaaf.png)

For cluster 0008 double click on 'Current level' and set attributes to 1/300/1:
![image](https://user-images.githubusercontent.com/28068256/74590110-c0ef6c00-500b-11ea-8e67-19fd70464442.png)

Viola!
